docs/configuration/alertmanager-integration/grafana-alert-manager.rst

```diff
@@ -1,25 +1,98 @@
 Grafana AlertManager
 ****************************************
 
-Grafana can send alerts to Robusta for automatic enrichment and visualization.
+Grafana can send alerts to the Robusta timeline for visualization and AI investigation.
 
 .. image:: /images/grafana-docs-robusta-ui.png
   :width: 600
   :align: center
 
 
-This guide only covers integrating alerts from Grafana Alerting with Robusta, not configuring Robusta to query metrics from the relevant Grafana data source.
+This guide only covers integrating alerts from Grafana Alerting with Robusta.
```

Comment:
After the edits, this sentence has lost all its meaning. (It only covers X as opposed to ???)

I suggest either removing it or replacing with 

This guide only covers sending alerts from Grafana Alerting to the Robusta timeline. If you'd like Robusta to also query metrics from Grafana, refer to general :ref:`metrics-integration docs for Prometheus <Integrating with Prometheus>`.

Reply:
fixed